html-forms/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> Html Forms</title>
+  <body>
+    <h1> Coding Form!
+    </h1>
+    <p>Email <br>
```

Comment:
Hi Evan! Please use label elements for your form titles. For example:

< label for="email-address" > Email </ label >